test/action/iterator.go

```diff
@@ -0,0 +1,99 @@
+package action
+
+import (
+	"github.com/sourcenetwork/corekv"
+	"github.com/sourcenetwork/corekv/test/state"
+	"github.com/stretchr/testify/require"
+)
+
+// Iterate action will create an iterator, execute the given child actions on the iterator,
```

Comment:
typo?

```suggestion
// Iterator action will create an iterator, execute the given child actions on the iterator,
```

Reply:
question: This is just the `struct` so is a bit confusing why the action stuff is included here.

Reply:
Thanks Shahzad, I've renamed `Actions` to `ChildActions`, fixed the typo and clarified that last `actions` word in the documentation.